src/components/molecules/Visualizer/Engine/Cesium/hooks.ts

```diff
@@ -172,12 +172,18 @@ export default ({
   }, []);
 
   // call onCameraChange event after moving camera
+  const emittedCamera = useRef<Camera[]>([]);
```

Comment:
The comment above this on 174 is specifically talking about the useCallback. Either move it to stay with the useCallback, or please update this comment to explain emittedCamera as well.

Reply:
This comment (and its place) comes from the earlier code but yes i should update it. 
It will be updated to an explaination to `emittedCamera`.